pytorch_lightning/trainer/model_hooks.py

```diff
@@ -17,6 +17,8 @@ def is_overridden(self, method_name: str, model: LightningModule = None) -> bool
             model = self.get_model()
         super_object = LightningModule
 
+        # assert model, 'no model passes'
```

Comment:
just crossed this, what is the case we would pass `None` as `model`?

Reply:
it is happening also in validation

Reply:
test(model=None) will take the model the trainer knows from fit, right?

Reply:
yes but in that walk-around test, the model was not assigned to trainer...